packages/rollup-full-node/test/app/routing-handler.spec.ts

```diff
@@ -80,14 +80,15 @@ const readOnlyPayload: JsonRpcSuccessResponse = {
   result: readOnlyResponse,
 }
 
-describe('Routing Handler', () => {
```

Comment:
Looks like there's an only here!

Reply:
Haha, yeah, I removed it while you were reviewing 😄 . Nice catch!